src/TpetraLinearSystem.C

```diff
@@ -2218,33 +2220,24 @@ int getDofStatus_impl(stk::mesh::Entity node, const Realm& realm)
       return DS_SharedNotOwnedDOF;
```

Comment:
I'm confused by "if owned return shared". Looks like this already existed before your commit.